### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by mredigonda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,60 +2,61 @@
...
-5. Go to repo `cd os4-react/client`
-6. Install dependences `npm install`
-7. Rebuild node-sass `npm rebuild node-sass`
-8. Run `gulp dev`
-9. Go to the main app: `http://localhost:3000/app` or the component demo `http://localhost:3000/demo`
-10. Your browser will automatically be opened and directed to the browser-sync proxy address
-12. Use `gulp dev --api` to disable fixtures and use the real php server api (it must be running at :8080).
+4. Install npm: `sudo apt-get install npm`
+5. Install gulp: `sudo npm install -g gulp`
+6. Go to client: `cd os4-react/client`
```
Comment (ivandiazwm) [inline README.md]: I think we should change it to `cd opensupports/client`, same goes for line 53 and 87

Comment (mredigonda) [reply README.md]: Agree. Fixed now.

File: server/API_STANDARD.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ OpenSupports can work completely fine without a frontend. You could only use thi
 This documentation is intended for application developers who are looking to integrate OpenSupports with other applications.
 
 ## Request/Response Interface
-The API server is located in the `api/` folder of your OpenSupports' instance folder.
+The API server is located in the `server/` folder of your OpenSupports' instance folder.
```
Comment (ivandiazwm) [inline server/API_STANDARD.md]: Revert this. OpenSupports builded package uses `api/` and requests should be made to that place

File: README.md [hunk 3]
```diff
@@ -66,41 +67,41 @@ Just as there is the `gulp dev` task for development, there is also a `gulp prod
...
-2. Install mysql dev dependencies `sudo apt-get install libmysqlclient-dev libmysqlclient16 ruby-dev`
-3. Install bundle `sudo gem install bundler`
-4. Go to test folder `cd os4-react/tests`
-5. Install project dependencies `sudo gem install bundler`
-Test can run by using executing `run-tests.sh` file.
+1. Install Ruby: `sudo apt-get install ruby-full`
+2. Install MySQL dev dependencies: `sudo apt-get install libmysqlclient-dev libmysqlclient16 ruby-dev`
+3. Install bundle: `sudo gem install bundler`
+4. Go to tests folder: `cd opensupports/tests`
+5. Install project dependencies: `sudo gem install bundler`
```
Comment (ivandiazwm) [inline README.md]: it should be `bundle install`

**Pull request merged**